api/RTCPeerConnection.json

```
@@ -1633,6 +1633,58 @@
             "deprecated": false
           }
         },
+        "Legacy_getStats": {
+          "__compat": {
+            "description": "Legacy (callback-based) <code>getStats()</code> method",
```

I don't know that "legacy" is particularly descriptive. What do you think of something like this?

```suggestion
            "description": "With callbacks (<code>getStats(selector, successCallback, failureCallback)</code>)",
```

---

We have a history of using the term "legacy" to describe something that only exists (if at all) for backward-compatibility reasons, or to give developers time to update code to use a newer API. I would prefer to leave this text as-is.

---

And in the end I wound up changing this to "Uses callbacks instead of promises"